modules/ROOT/partials/configuration/valid_elements.adoc

```diff
@@ -69,10 +71,11 @@ Wildcards such as `*`,`+`,`?` may be used in element or attribute name matching.
 | Results in a unique ID. For example, `'p[id:{$uid}]'`.
 |===
 
-Use `*[*]` to include all elements and all attributes. This can be very useful when used with the link:{baseurl}/configure/content-filtering/#invalid_elements[invalid_elements] option.
+Use `*[*]` to include all elements and all attributes. This can be very useful when used with the xref:invalid_elements[`invalid_elements`] option.
```

Comment:
This isn't rendering correctly... it's rendering the first bracket as bold.
```suggestion
Use `+*[*]+` to include all elements and all attributes. This can be very useful when used with the xref:invalid_elements[`invalid_elements`] option.
```